argus/recipes/cloud/windows.py

```diff
@@ -203,6 +203,19 @@ def replace_code(self):
         command = '"{}" -m pip install -r C:\\cloudbaseinit\\requirements.txt'
         self._execute(command.format(python))
 
+    def _add_localscripts_file(self):
+        """Add various scripts to the localscripts folder,
+        in order to later test if the cloudbase plugin is
```

Comment:
I think it should be: `in order to later test if the cloudbase-init localscripts plugin is`